package.json

```diff
@@ -18,6 +18,7 @@
         "node-gyp": "^7.1.0",
         "node-sass": "^4.0.0",
         "react": "^16.13.1",
+        "react-cookie-consent": "^5.1.3",
```

Comment:
why have you added this? see the user story

Reply:
This is like jquery but in the react version. I used the library for React.

Reply:
there's no need for this. remove it and checkout the jira ticket. everything is there on what's needed. or sync up with @vasynizami